MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.md
+include cadquery/py.typed
```

Comment:
I thought it was strange that MANIFEST was so different from MANIFEST.in so I put it there. Another otpion is it can be in `setup.py` as `package_data={"cadquery": ["py.typed"]},`. [See this](https://mypy.readthedocs.io/en/stable/installed_packages.html?highlight=pip%20561#making-pep-561-compatible-packages).

Reply:
Sure, either way MANIFEST (but not MANIFEST.in) file should be removed AFAIK. I'm not a python packaging expert, just followed https://blog.ionelmc.ro/presentations/packaging/#slide:1

Reply:
I'm a total neophyte and never packaged anything other than a toy package in a tutorial, LoL.